CRM/Admin/Form/Preferences/Mailing.php

```diff
@@ -106,7 +106,13 @@ public function preProcess() {
           'html_type' => 'checkbox',
           'title' => ts('Hashed Mailing URL\'s'),
           'weight' => 11,
-          'description' => 'If enabled, a randomized hash key will be used to reference the mailing URL in the mailing.viewUrl token, instead of the mailing ID',
+          'description' => ts('If enabled, a randomized hash key will be used to reference the mailing URL in the mailing.viewUrl token, instead of the mailing ID'),
+        ),
+        'auto_recipient_rebuild' => array(
+          'html_type' => 'checkbox',
+          'title' => ts('Enable automatic CiviMail recipient count display'),
+          'weight' => 12,
+          'description' => ts('Enable this setting to rebuild recipient list automatically during composing mail. Disable will allow you to rebuild recipient manually.'),
```

Comment:
@mlutfy I have added ts(). Thanks for notifying :)  Is there anything else that needs to be done to get this PR merged?